src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -13,8 +13,11 @@ public class DoctorConstants {
     public static final String CREATE_DOCTOR_END_POINT = "/saveDoctors";
     public static final String DEPARTMENT_SHOULD_BE_MANDATORY = "Department name should be mandatory!";
```

Comment:
Approved. But, try to maintain consistency with the code. Better to have all the error and validation messages in the messages.properties file.

Reply:
done all the validations have been moved to message.properties